### PR TITLE
Feat wishlist

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
Added wishlist components, changed products and added contexts for wishlist

### CODE REVIEW

**Review by Avin008 — APPROVED** [review]
well structured code, looks good. good to merge

File: src/components/CartCard.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <article className="prod-in-cart">
+      <div className="image">
+        <img src={img} alt={title} />
+      </div>
+      <div className="prod-detail">
+        <div className="text text-align-left">
+          <p className="large-text">{title}</p>
+          <p className="large-text">
+            Rs.{price} 
+            {/* <span className="text-lin-through grey-text">Rs.{price}</span> */}
```
Comment (Tanjum786) [inline src/components/CartCard.jsx:14]: remove commented code

**Review by Tanjum786 — APPROVED** [review]
code looks clean and understandable 
just a small suggestion mention brief summary of PR in the description that the reviewer can understand that  what the PR contain

File: src/components/CartCard.jsx (hunk 1 above)
Comment (Kothalkarpriya) [reply src/components/CartCard.jsx:14]: Ok I will

**Pull request merged**